lib/political_content_identifier.rb

```diff
@@ -0,0 +1,36 @@
+class PoliticalContentIdentifier
+  POLITICAL_PUBLICATION_TYPES = [
+    PublicationType::ImpactAssessment,
+    PublicationType::InternationalTreaty,
+    PublicationType::PolicyPaper,
+    PublicationType::ResearchAndAnalysis,
+    PublicationType::CorporateReport
```

Comment:
As per the style guide can this have a trailing comma just incase we add another type for a cleaner diff. Also is this alphabetical except for corporate report or is that just coincidental?